Cargo.toml

```diff
@@ -7,9 +7,10 @@ license = "MIT/X11 OR Apache-2.0"
 documentation = "https://docs.rs/confy"
 repository = "https://github.com/rust-cli/confy"
 readme = "README.md"
-edition = "2018"
+edition = "2021"
```

Comment:
Thanks for doing this! Could you revert this change though, I'd like to bump the edition separately. 

If you need it for this MR I can do so before this merges :)

Reply:
Resolved in https://github.com/rust-cli/confy/pull/71/commits/31192c9fd87809ef76a323c74d99e54e127ab48c